include/picongpu/particles/atomicPhysics/rateCalculation/BoundFreeFieldTransitionRates.hpp

```diff
@@ -39,77 +40,108 @@ namespace picongpu::particles::atomicPhysics::rateCalculation
     template<atomicPhysics::enums::ADKLaserPolarization T_ADKLaserPolarization>
     struct BoundFreeFieldTransitionRates
     {
-        /** rate for bound-free field ionization transition of ion depending on the local electric field
-         *
-         * @tparam T_ChargeStateDataBox instantiated type of dataBox
-         * @tparam T_AtomicStateDataBox instantiated type of dataBox
-         * @tparam T_BoundFreeTransitionDataBox instantiated type of dataBox
+    private:
+        struct ResultStruct
```

Comment:
```suggestion
        struct RateFormulaVariables
```
or something else but the original is very misleading.

Reply:
done